providers/dns/zoneee/client.go

```diff
@@ -0,0 +1,132 @@
+package zoneee
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"path"
+)
+
+const defaultEndpoint = "https://api.zone.eu/v2/dns"
```

Comment:
It's missing a `/` at the end. Otherwise the `dns` gets replaced with the domain we want to access and the call fails.

Reply:
nice catch :+1: